content/STRUCTURE.json

```diff
@@ -330,6 +355,16 @@
      "generator": "link",
      "meta": {"url": "https://github.com/lxc/distrobuilder"}},
 
+    {"path": "/distrobuilder/github-actions/",
+     "menu": ["distrobuilder", "GitHub Actions"],
+     "generator": "link",
+     "meta": {"url": "https://github.com/lxc/distrobuilder/actions"}},
+
+    {"path": "/distrobuilder/jenkins-ci/",
+     "menu": ["distrobuilder", "Jenkins CI"],
+     "generator": "link",
+     "meta": {"url": "https://jenkins.linuxcontainers.org/view/Distrobuilder/"}},
```

Comment:
Same as LXCFS, there's no meaningful tests being run on Jenkins for distrobuilder.